gpflow/kernels/periodic.py

```diff
@@ -32,35 +35,49 @@ class Periodic(Kernel):
     γ is the period parameter.
 
     (note that usually we have a factor of 4 instead of 0.5 in front but this
-    is absorbed into lengthscale hyperparameter).
+    is absorbed into the lengthscale hyperparameter).
     """
 
-    def __init__(self, base: Stationary, period: Union[float, List[float]] = 1.0):
+    def __init__(self, base: Stationary, period: Union[float, List[float]] = 1.0,
```

Comment:
I'm not sure why we need to handle the active dims argument here. why can't we just not _have_ the active dims argument, but respect the one in the base kernel?

Reply:
That's how it was before. But it confused users (https://github.com/GPflow/GPflow/issues/1230), and the `Kernel` base-class defines active dims as part of the kernel interface - and all child classes should have a consistent interface. This makes it easier for users, and they get a clear error message when they try and set active_dims to something inconsistent with base.active_dims.